### PR TITLE
fix(learn) : Updated the regex in test 

### LINKED ISSUE #39687: Basic JavaScript: Passing Values to Functions with Arguments
**Describe your problem and how to reproduce it:**

The challenge instructions say:
> 2. Call the function with two numbers as arguments.

But you can only pass positive integers. I suggest that either the instruction is changed to:
> 2. Call the function with two positive integers as arguments.
or the regex should be changed to:
```
^\s*functionWithArgs\s*\(\s*[-+]?[0-9]*\.?[0-9]*.\s*,\s*[-+]?[0-9]*\.?[0-9]*.\s*\)\s*`
```
So that it can accept negatives and decimals.

Changing the regex would probably be simpler because it could be changed across all the language files without having to worry about translation.

I can handle this once there is agreement.

**Add a Link to the page with the problem:**

[here](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments)

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: Version 85.0.4183.121 (Official Build) (64-bit)
* Operating System: Macbook Catalina

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

Screen Shot 2020-09-24 at 9 34 38 PM
Screen Shot 2020-09-24 at 9 35 25 PM

### PR DESCRIPTION
**Pull request by mainak-debnath**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39687 

Updated the regex in test so that the function also work with negative numbers.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
LGTM~! 🥳 

Thank you for your contributions to freeCodeCamp!

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments.english.md [hunk 1]
```diff
@@ -39,8 +39,7 @@ tests:
   - text: <code>functionWithArgs(7,9)</code> should output <code>16</code>.
     testString: if(typeof functionWithArgs === "function") { capture(); functionWithArgs(7,9); uncapture(); } assert(logOutput == 16);
   - text: You should call <code>functionWithArgs</code> with two numbers after you define it.
-    testString: assert(/^\s*functionWithArgs\s*\(\s*\d+\s*,\s*\d+\s*\)\s*/m.test(code));
-
+    testString: assert(/functionWithArgs\s*\(\s*[-+]?[0-9]*\.?[0-9]*.\s*,\s*[-+]?[0-9]*\.?[0-9]*.\s*\)\s*/.test(code.replace(/\s/g, '')));
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments.english.md:42]:
```suggestion
    testString: assert(/functionWithArgs\([-+]?\d*\.?\d*.,[-+]?\d*\.?\d*.\)/.test(code.replace(/\s/g, '')));
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
There is no need to check for spaces if we use `replace` on the code. Also, `\d` is clear enough.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments.english.md [hunk 2]
```diff
@@ -39,8 +39,7 @@ tests:
   - text: <code>functionWithArgs(7,9)</code> should output <code>16</code>.
     testString: if(typeof functionWithArgs === "function") { capture(); functionWithArgs(7,9); uncapture(); } assert(logOutput == 16);
   - text: You should call <code>functionWithArgs</code> with two numbers after you define it.
-    testString: assert(/^\s*functionWithArgs\s*\(\s*\d+\s*,\s*\d+\s*\)\s*/m.test(code));
-
+    testString: assert(/functionWithArgs\([-+]?\d*\.?\d*.,[-+]?\d*\.?\d*.\)/.test(code.replace(/\s/g, '')));
```
Comment (Sky020) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/passing-values-to-functions-with-arguments.english.md:42]:
```suggestion
    testString: assert(/functionWithArgs\([-+]?\d*\.?\d*,[-+]?\d*\.?\d*\)/.test(code.replace(/\s/g, '')));
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Sorry, but I missed this. There is no reason for the un-escaped `.`

I could fully expect a camper to do this (and accidentally pass):
```
functionWithArgs(a, b);
```

**Comment by mainak-debnath** [comment]
@Sky020 Is this ok?

**Review by ShaunSHamilton — APPROVED** [review]
Thanks for that, @mainak-debnath . This LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]
LGTM! 🥳 

Thank you for contributing to freeCodeCamp!

**Comment by Sky020** [comment]
@mainak-debnath , for this pull request (PR)! 

Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**